docs/UserGuide.md

```diff
@@ -273,6 +273,11 @@ To find flashcards, you can find them by specifying keywords within their questi
 
 :information_source: You must provide at least one search parameter.
 
+<div markdown="block" class="alert alert-danger">
+:exclamation: Take note that only flashcards which fullfill **all** the specified search parameters will be displayed. Flashcards that fullfill only some search parameters will not be diplayed.
```

Comment:
think there is a small typo here, should `fulfill` instead of `fullfill`

Reply:
Okay fixed it!